sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/Field.java

```diff
@@ -314,7 +326,7 @@ public Field setKey(Boolean key) {
      *
      * @return the retrievable value.
      */
-    public Boolean isRetrievable() {
+    private Boolean isRetrievable() {
```

Comment:
Do we need these private getters and setters at all? We can just set the member variable.

Reply:
Oh. I see. This is generated

Reply:
This is auto-gen code. I hope there is an easy way to remove getter and setter. Unfortunately, we have to write tons of codes in transformation. It is not a huge burden to leave it here. Try to balance the maintenance of transformation file and model structure here.